inception/inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/config/StringMatchingRecommenderAutoConfiguration.java

```diff
@@ -82,7 +81,6 @@ public StringMatchingRecommenderFactory stringMatchingRecommenderFactory(
     }
 
     @Bean
-    @ConditionalOnProperty(prefix = "recommender.string-matching.relation", name = "enabled", havingValue = "true")
```

Comment:
Why did you remove the conditional?

Reply:
I think its time for them to be enabled by default in 21

Reply:
I think that deserves a separate PR, no? Also the documentation probably needs to be updated then to remove the "experimental" notice there.

Reply:
I can do that

Reply:
I believe pulling out the change to enable the relation recommender by default is the only "blocker" for merging this PR, right?

Reply:
I wanted to wait for feedback from the guy, but yes, that is the only blocker if you think its ready to be merged otherwise.

Reply:
I undid it